In message <20030225131546.GL7685@fs.tum.de>,Adrian Bunk writes:
>drivers/atm/fore200e.o drivers/atm/fore200e.c
>drivers/atm/fore200e.c: In function `fore200e_push_rpd':
>drivers/atm/fore200e.c:1135: structure has no member named `timestamp'
>drivers/atm/fore200e.c:1136: structure has no member named `timestamp'

it shouldnt be doing that.  you only need to set the timestamp in the 
skb.  i see the eni driver does the same thing.  i will see about
a patch for that shortly.

Index: linux/drivers/atm/fore200e.c
===================================================================
RCS file: /home/chas/CVSROOT/linux/drivers/atm/fore200e.c,v
retrieving revision 1.1.1.1
diff -u -d -b -w -r1.1.1.1 fore200e.c
--- linux/drivers/atm/fore200e.c	20 Feb 2003 13:45:03 -0000	1.1.1.1
+++ linux/drivers/atm/fore200e.c	25 Feb 2003 14:42:06 -0000
@@ -1135,7 +1135,7 @@
 	return;
     } 
 
-    skb->stamp = vcc->timestamp = xtime;
+    do_gettimeofday(&skb->stamp);
     
 #ifdef FORE200E_52BYTE_AAL0_SDU
     if (cell_header) {
-
To unsubscribe from this list: send the line "unsubscribe linux-kernel" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html
Please read the FAQ at  http://www.tux.org/lkml/
